meson.build

```diff
@@ -0,0 +1,30 @@
+project('list-sensors', 'cpp',
+    default_options: [
+        'warning_level=3',
+        'werror=true',
+        'cpp_std=c++17',
+    ],
+    license: 'Apache-2.0',
+    version: '2.0',
+)
+
+conf = configuration_data()
+
+conf.set_quoted('MAPPER_BUS', get_option('mapper-bus'))
+conf.set_quoted('MAPPER_PATH', get_option('mapper-path'))
+conf.set_quoted('MAPPER_IFACE', get_option('mapper-iface'))
+conf.set_quoted('SYSTEMD_PROPERTIES', get_option('systemd-properties'))
```

Comment:
It's not a systemd specific. It is a standard D-Bus interface, actually you even can't change it, so this option is always a constant and can not be changed.

Reply:
Moved to source as constant.